x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/endpoint_details_tabs.tsx

```diff
@@ -67,25 +86,18 @@ export const EndpointDetailsFlyoutTabs = memo(
             },
           });
         }
-        return setSelectedTabId(tab.id as EndpointDetailsTabsId);
       },
-      [dispatch, pageSize, setSelectedTabId]
+      [dispatch, pageSize]
     );
 
-    const selectedTab = useMemo(() => tabs.find((tab) => tab.id === selectedTabId), [
-      tabs,
-      selectedTabId,
-    ]);
+    const selectedTab = useMemo(() => tabs.find((tab) => tab.id === show), [tabs, show]);
 
     const renderTabs = tabs.map((tab) => (
-      <EuiTab
-        onClick={() => handleTabClick(tab)}
-        isSelected={tab.id === selectedTabId}
-        key={tab.id}
-        data-test-subj={tab.id}
-      >
-        {tab.name}
-      </EuiTab>
+      <EndpointDetailsTab
+        tab={tab}
+        handleTabClick={() => handleTabClick(tab)}
+        isSelected={tab.id === show}
```

Comment:
Should this instead be `tab.id === selectedTab.id`? (it probably does not matter :) )

Reply:
done 8253ccdd9e4bcb06de841758ad2166f4a2c0b779